client/src/components/restaurant/restaurant.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import { Tab, Tabs } from 'react-bootstrap';
+import Api from '../../services/api';
+import About from './about';
+
+class Restaurant extends Component {
+  getAll() {
+    Api.getAll();
+  }
+
+  render() {
+    return (
+      <div>
+        <h2>Title</h2>
+        <Tabs defaultActiveKey="about">
+          <Tab eventKey="about" title="About">
+            <About />
```

Comment:
We use tab?

Reply:
Yes. According to design_decisions and mock-ups channels